schemas/context.json

```diff
@@ -42,6 +42,39 @@
 				"nullable"
 			]
 		},
+        "enableDois": {
+            "type": "boolean",
+            "default": 0,
```

Comment:
The current Doi plugin is set to disabled by default. Would it be better to keep the current default? If it is enabled by default, should any of the pub object types (like articles) also be enabled by default? I can imagine it may cause confusion if DOIs (as a setting) were enabled by default but none of the types were. Articles feels like a sensible default for OJS, but do you have a sense of what this would be for OMP?

Reply:
Hmm, yeah that's interesting. Well ok, if it wasn't enabled by default before let's not do it. We might change that later but we can think it through more.